layouts/partials/navbar.html

```diff
@@ -15,38 +15,79 @@
   <nav class="nav">
     <ul class="nav__list" id="navMenu">
       {{ $url := .RelPermalink }}
+      {{ $current := . }}
       {{ range .Site.Menus.main }}
         {{ $active := eq $url .URL }}
-        <li class="nav__list-item">
-          <a
-            {{ if $active }}
-              class="nav__link--active"
-
-            {{ end }}
-            href="{{ .URL }}"
-            {{ if strings.HasPrefix .URL "http" }}
-              target="_blank" rel="noopener noreferrer"
-
-            {{ end }}
-            title="{{ .Title }}"
-            >{{ .Name }}</a
-          >
-        </li>
+        {{ if .HasChildren }}
+          <li class="nav__list-item">
+            <div class="optionswitch">
+              <input class="optionswitch__picker" type="checkbox" id="menuoptionpicker" hidden />
+              <label
+                {{ range .Children }}
+                  {{ if in $current.RelPermalink .URL }}
```

Comment:
You should add an else clause, as otherwise the cursor won't be displayed properly if someone hovers over the menu. This could look like the following.

```
{{ else}}
  class="optionswitch__label"
{{ end }}
```